src/store/modules/Settings/NetworkStore.js

```diff
@@ -248,6 +334,44 @@ const NetworkStore = {
           );
         });
     },
+    async updateIpv6Address({ dispatch, state }, newIpv6Address) {
+      const originalAddresses =
+        state.networkSettings[state.selectedInterfaceIndex].staticIpv6Addresses;
+      const updatedIpv6 = originalAddresses.map((item) => {
+        const address = item.Address;
+        console.log(find(newIpv6Address, { Address: address }));
```

Comment:
I think we can remove this

Reply:
console.log(find(newIpv6Address, { Address: address }));

Reply:
Done